handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Restart mattermost-github-integration service
+  service:
+    name: mattermost-github-integration
+    enabled: yes
+    state: restarted 
+  listen: "restart service"
```

Comment:
Why do you even have this listen? Just do `notify: Restart mattermost-github-integration service` and remove the listen

Reply:
fixed